.copier-answers.yml

```diff
@@ -1,11 +1,12 @@
 # Do NOT update manually; changes here will be overwritten by Copier
-_commit: v1.0.2
+_commit: v1.1.1
 _src_path: gh:oca/oca-addons-repo-template
 dependency_installation_mode: PIP
+generate_requirements_txt: true
 include_wkhtmltopdf: false
 odoo_version: 14.0
 rebel_module_groups: []
-repo_description: "TODO: add repo description."
+repo_description: 'TODO: add repo description.'
```

Comment:
@sbidoul It's curious that this is change, but afterwards pre-commit fails because of it. Maybe the template is incorrect and include single quote instead of double?

Reply:
Yes the normal process is `copier -f update && pre-commit run -a` which automatically fixes these issues.
Here copier update created a `.rej` file so my script stopped on that and did not run pre-commit run -a.x

But yes, changing the quotes in the template can't hurt.

Reply:
> But yes, changing the quotes in the template can't hurt.

Ah, no this file is generated by copier itself, maybe we should add it to the pre-commit exclude list.

Reply:
Or do it pre-commit friendly. cc @Yajo

Reply:
The real problem is the `.rej` file below, as @sbidoul said. There's a merge conflict, thus that requires manual intervention.

This problem with quotes would be fixed with pre-commit automatically after the conflicts are fixed, so not much to do here.

The YAML is dumped with PyYAML, which doesn't always dump the prettiest YAML, so the easiest fix is anyways to run pre-commit later.